lib/stat_tracker.rb

```diff
@@ -240,10 +240,24 @@ def fewest_tackles #issue # 22
 
   end
 
-  def team_info #issue # 23
-
-
-
+  def team_info(team_id) #issue # 23
+
+      info = {
+      "team_id" => team_id,
+      "franchise_id" => 0,     
+      "team_name" => 0,
+      "abbreviation" => 0, 
+      "link" => 0
+              }
+
+      @teams.each do |row|
+        info["franchise_id"] = row[:franchiseid].to_s if row[:team_id] == team_id.to_i
+        info["team_name"] = row[:teamname] if row[:team_id] == team_id.to_i
+        info["abbreviation"] = row[:abbreviation] if row[:team_id] == team_id.to_i
+        info["link"] = row[:link] if row[:team_id] == team_id.to_i
+      end
+      
```

Comment:
The way you did this `each` and called on the info by `row` first vs `header` first inspired me on a way to make two of my helper methods easier to read and use on issue #18 .